modules/cms/models/ThemeData.php

```diff
@@ -125,7 +126,11 @@ public function afterFetch()
                 $this->jsonable[] = $id;
             }
             elseif ($field['type'] === 'fileupload') {
-                $this->attachOne[$id] = File::class;
+                if (str_contains($field['mode'], '-multi')) {
```

Comment:
@mjauvin Might be better to make this more explicit:
```suggestion
                if (in_array($field['mode'], ['file-multi', 'image-multi'])) {
```